src/app/share-preview/components/share-preview/share-preview.component.ts

```diff
@@ -22,7 +22,7 @@ import {
 } from '@shared/utilities/forms';
 import { AccountResponse, AuthResponse } from '@shared/services/api/index.repo';
 import { DeviceService } from '@shared/services/device/device.service';
-import { Subscription } from 'rxjs';
+import { Subscription, Subject } from 'rxjs';
```

Comment:
What does this new import do?

Reply:
(just curious as ever)

Reply:
`Subject` is an object that can be subscribed to in RxJS, and this is used so the share banner component can subscribe to this subject and receive events from its parent. This is apparently the "proper" way to do this kind of parent-child event communication in Angular!